src/revaultd/model.rs

```diff
@@ -152,6 +152,16 @@ impl VaultStatus {
     ];
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub enum SpendTxStatus {
+    #[serde(rename = "non_final")]
+    NonFinal,
+    #[serde(rename = "immature")]
+    Immature,
```

Comment:
I think it is `Pending` https://github.com/revault/revaultd/pull/213/commits/3a75404b5516846957aafeb5cb3200496ae92d7c#diff-d5a5ac14f4498f3c9a33ba44deb27be25ffe35fe57dcea34fe7904f728955e31R133

Reply:
Thanks, I forgot I called them like that... it's a bit confusing that what's "pending" in the ui is not "pending" in the code (:sob:), but yeah, my fault